packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -133,8 +134,8 @@ export class SchematizingSimpleTreeView<
 		this.update();
 
 		this.unregisterCallbacks.add(
-			this.checkout.events.on("commitApplied", (data, getRevertible) =>
-				this.events.emit("commitApplied", data, getRevertible),
+			this.checkout.events.on("changed", (data, getRevertible) =>
+				this.events.emit("changed", data, getRevertible),
```

Comment:
we still need to fire "commitApplied" as well, right?

Reply:
And retain test coverage until it's actually removed

Reply:
oh right, yes

Reply:
so there weren't any tests specifically testing this event, just ones related to undo, so I added an undo test for just the commitApplied event